### ISSUE TITLE
Add sampling of "sites next to each other" to genes

### ISSUE THREAD

**Issue by redavids** [hyaduvanshi/quartetstotree#1]
Currently, we have the ability to sample k sites (columns) independently from a alignment (phylip file).  This is a request for an OPTION and not a REPLACEMENT: To do a comparison with summary methods, we really should be taking k contiguous columns-but the location of the contiguous columns should be random.  E.g., once the beginning of the k columns to be selected is randomly chosen, then take the next k-1 columns.  

Please contact redavids@ncsu.edu with questions

**Comment by agupta0905** [comment]
Ok am and shashank will look into that and add that option

**Comment by redavids** [comment]
Thanks!!!!

On Mon, Apr 6, 2015 at 2:32 PM, Ashu Gupta <notifications@github.com> wrote:

> Ok am and shashank will look into that and add that option
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hyaduvanshi/quartetstotree/issues/1#issuecomment-90207559>
> .
>



-- 
Ruth Davidson
Department of Mathematics, University of Illinois Urbana-Champaign
http://www.math.uiuc.edu/~redavid2/

**Comment by hyaduvanshi** [comment]
Have added this in Ashu's github which is where our combined scripts are.
Ashu will now add this in the shell script too.

**Comment by redavids** [comment]
Thanks a lot you guys!  I was under the impression we were going to keep everything on this github as we discussed last meeting-can you move those things here?  Or move everything here there?

**Comment by hyaduvanshi** [comment]
Yes so instead of combining everything here, we will just keep everything there. All our relevant stuff is already there, Jed can add stuff also there.
